internal/pkg/model/object.go

```diff
@@ -14,9 +14,9 @@ import (
 )
 
 const (
-	MetaFileTag                       = "metaFile:true"
-	ConfigFileTag                     = "configFile:true"
-	DescriptionFileTag                = "descriptionFile:true"
+	MetaFileFieldsTag                 = "metaFile:true"        // marks meta fields in object struct
```

Comment:
Tieto tagy su `Go` tags pre struktury objektov, tak som im trocha upravil nazov, aby to bolo viac zrejme.